src/site/_ru/_betterbook.yaml

```diff
@@ -57,9 +57,9 @@ toc:
         path: /fundamentals/discovery-and-distribution/
         id: discovery-and-distribution
         section:
-          - title: "Optimization for Crawlers"
+          - title: "Оптимизация под поисковых ботов"
             path: /fundamentals/discovery-and-distribution/optimization-for-crawlers/
-          - title: "Enriching Search Results"
+          - title: "Улучшение результатов поиска"
```

Comment:
@PaulKinlan Do I get it right that by "enriching" you understand some good practices to follow when marking up the page so there appear more chances of the considered website to be found and placed as above as possible in the search result list?